packages/react-swipeable-views/src/SwipeableViews.js

```diff
@@ -746,6 +756,17 @@ class SwipeableViews extends Component {
     const touchEvents = disabled ? {} : {
       onTouchStart: this.handleTouchStart,
       onTouchEnd: this.handleTouchEnd,
+      onMouseDown: (e) => {
+        e.preventDefault();
+        e.persist();
```

Comment:
I was getting a react warning about modifying a shared event otherwise. This stopped the warning from appearing but maybe there is a better way.